extensionsGallery.json

```diff
@@ -3090,14 +3090,14 @@
 					},
 					"versions": [
 						{
-							"version": "0.2.1",
-							"lastUpdated": "06/03/2020",
+							"version": "0.2.3",
```

Comment:
I'd suggest reverting this change - I'll update my open PR for updating the stable gallery (unless you want to get it out ASAP)